docs/resources/azure_express_route_circuit.md

```diff
@@ -0,0 +1,124 @@
+---
+title: About the azure_express_route_circuit Resource
+platform: azure
+---
+
+# azure_express_route_circuit
+
+Use the `azure_express_route_circuit` InSpec audit resource to test properties related to a express circuit resource.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`resource_group` and express circuit resource `name` or the `resource_id` must be given as a parameter.
+```ruby
+describe azure_express_route_circuit(resource_group: 'MyResourceGroup', name: 'express circuit_name') do
+  it { should exist }
+end
+```
+## Parameters
```

Comment:
these are properties and not parameters and here parameters are actually missing

Reply:
corrected.